examples/shared_samples/server_certificate.rb

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require_relative '../_client' # Gives access to @client
+
+# NOTE: You'll need to add the following instance variable to the _client.rb file with valid values for your environment:
+#   @storage_system_ip
+
+# All supported APIs for  Server Certificate:
+# - 600, 800, 1000, 1200
+# # Resources classes that you can use for Client Certificate in this example:
```

Comment:
its server certificate. please correct it everywhere

Reply:
@soodpr Corrected